python/nano/tutorial/inference/pytorch/neural_coder/example.py

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+
+import torch
+from torchvision.models import resnet18
+
+if __name__ == "__main__":
+
+    model_ft = resnet18(pretrained=True)
+
+    x = torch.rand(2, 3, 224, 224)
+    y_hat = model_ft(x)
```

Comment:
If you only inference once, the effect of each acceleration method may not be obvious.
Can we add a loop like `for i in range (100)`?

Reply:
could not do this since NC still have some issue for this mode of calling.